test/Api.Test/Utilities/AccessPolicyHelpersTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Bit.Api.SecretsManager.Utilities;
+using Bit.Core.Exceptions;
+using Bit.Core.SecretsManager.Entities;
+using Xunit;
+
+public class AccessPolicyHelpersTests
+{
+    [Fact]
+    public void CheckForDistinctAccessPolicies_ThrowsExceptionWhenDuplicateExists()
+    {
+        var duplicatePolicy = new UserProjectAccessPolicy
+        {
+            OrganizationUserId = Guid.NewGuid(),
+            GrantedProjectId = Guid.NewGuid()
+        };
+
+        var accessPolicies = new List<BaseAccessPolicy>
+            {
+                duplicatePolicy,
+                duplicatePolicy // Duplicate policy
+            };
+
+        Assert.Throws<BadRequestException>(() =>
+        {
+            AccessPolicyHelpers.CheckForDistinctAccessPolicies(accessPolicies);
+        });
+    }
+
+    [Fact]
+    public void CheckForDistinctAccessPolicies_Success()
+    {
+        var accessPolicies = new List<BaseAccessPolicy>
+            {
+                new UserProjectAccessPolicy
+                {
+                    OrganizationUserId = Guid.NewGuid(),
+                    GrantedProjectId = Guid.NewGuid()
+                },
+                new GroupProjectAccessPolicy
+                {
+                    GroupId = Guid.NewGuid(),
+                    GrantedProjectId = Guid.NewGuid()
+                }
+            };
+
+        try
+        {
+            AccessPolicyHelpers.CheckForDistinctAccessPolicies(accessPolicies);
+        }
+        catch (Exception ex)
+        {
+            Assert.True(false, $"Unexpected exception: {ex.Message}");
+        }
```

Comment:
```suggestion
      AccessPolicyHelpers.CheckForDistinctAccessPolicies(accessPolicies);
```

Xunit will handle any potential exceptions for us and fail the test if they occur.

Another option is to use the Record.Exception 

```
  var exception = Record.Exception(() => AccessPolicyHelpers.CheckForDistinctAccessPolicies(accessPolicies));

        Assert.Null(exception);
```